devel/fetch_metrics/fetch_metrics.py

```diff
@@ -96,7 +96,8 @@ def slurp(filename: str) -> str:
     return as_one_string
 
 def load_sortings() -> List[Dict[str, Any]]:
-    hydrated_sortings = json.loads(slurp(args['sortingsfile']))
+    # hydrated_sortings = json.loads(slurp(args['sortingsfile']))
+    hydrated_sortings = kp.load_json(args['sortingsfile'])
```

Comment:
Here's where we load sortingsfile from kachery - this also works for local file paths.